docs/usage.rst

```diff
@@ -69,6 +69,48 @@ callable is called, or to prevent it, use the ``call_default`` parameter:
     Note that ``call_default`` is only used when the related setting is missing
     from the project settings!
 
+Using environment variables
+'''''''''''''''''''''''''''
+
+Nowadays it became more and more popular to read settings from the environment.
+This functionality is supported as well. If the setting is found in environment, its
+value is used and takes precedence over any value present in the settings module.
+By default, all values are parsed as JSON, see other supported values in description
+of individual setting classes.
+
+Anyway, you can override the environ value parsing on your own. It is done by the
+``decode_environ(self, value)`` method.
+
+Example
+-------
+
+Let's assume following setting class, following django setting module and following
+illustration file:
+
+.. code:: python
+
+    # setting class
+    class MySettings(AppSettings):
+        example = StringSetting()
+        array = IterableSetting()
+        dictionary = DictSetting()
+        boolean = BooleanSetting(default=True)
+
+    # DJANGO_SETTING_MODULE
+    SECRET_KEY = "__MY_SUPER_SECRET__"  # required setting by Django itself
+    EXAMPLE = "example"
+    ARRAY = [1, 2, 3]
+    DICTIONARY = {"key": "value"}
+
+.. code:: python
+
+    $ ARRAY='abc:def:ghi' DICTIONARY='{"key": "val", "key2": "val2"}' python illustration.py
+    > example
+    > ['abc', 'def', 'ghi']
+    > {'key1': 'val1', 'key2': 'val2'}
+    > True
```

Comment:
I'd keep the example copy-pastable. I suggest

```python
import os
from appsettings import AppSettings, StringSetting

class MySettings(AppSettings):
    example = StringSetting()

os.environ['EXAMPLE'] = 'Example value'
settings = MySettings()
print(settings.example)  # >>> 'Example value'
```

Reply:
Done.